vars/longevityPipeline.groovy

```diff
@@ -20,92 +20,117 @@ def call(Map pipelineParams) {
             SCT_TEST_ID = UUID.randomUUID().toString()
             SCT_GCE_PROJECT = "${params.gce_project}"
         }
-        parameters {
+        options {
+            timestamps()
+            disableConcurrentBuilds()
+            buildDiscarder(logRotator(numToKeepStr: '20'))
+        }
+    stages {
+        stage('Parameters') {
+            steps {
+                script {
+                    properties([
+                        parameters([
+            separator(name: 'CLOUD_PROVIDER', sectionHeader: 'Cloud Provider Configuration'),
             string(defaultValue: "${pipelineParams.get('backend', 'aws')}",
                description: 'aws|gce|azure|docker',
-               name: 'backend')
+               name: 'backend'),
 
             string(defaultValue: "${pipelineParams.get('region', 'eu-west-1')}",
                description: 'Supported: us-east-1 | eu-west-1 | eu-west-2 | eu-north-1 | eu-central-1 | us-west-2 | random (randomly select region)',
-               name: 'region')
+               name: 'region'),
             string(defaultValue: "${pipelineParams.get('gce_datacenter', 'us-east1')}",
                    description: 'GCE datacenter',
-                   name: 'gce_datacenter')
+                   name: 'gce_datacenter'),
             string(defaultValue: "${pipelineParams.get('azure_region_name', 'eastus')}",
                    description: 'Azure location',
-                   name: 'azure_region_name')
+                   name: 'azure_region_name'),
             string(defaultValue: "${pipelineParams.get('availability_zone', 'a')}",
                description: 'Availability zone',
-               name: 'availability_zone')
+               name: 'availability_zone'),
 
+            // Stress Test Configuration
+            separator(name: 'STRESS_TEST', sectionHeader: 'Stress Test Configuration'),
             string(defaultValue: "",
                description: 'Duration in minutes for stress commands(gemini, c-s, s-b)',
-               name: 'stress_duration')
+               name: 'stress_duration'),
 
             string(defaultValue: "",
                description: ('Time duration in minutes for preparing dataset with commands prepare_*_cmd, if empty value, default value is 5h = 300 minutes.' +
                              'Prepare commands could finish earlier and have not to run full prepare_stress_duration time'),
-               name: 'prepare_stress_duration')
+               name: 'prepare_stress_duration'),
 
-            string(defaultValue: '', description: '', name: 'scylla_ami_id')
-            string(defaultValue: '', description: '', name: 'gce_image_db')
-            string(defaultValue: '', description: '', name: 'azure_image_db')
+            // ScyllaDB Configuration
+            separator(name: 'SCYLLA_DB', sectionHeader: 'ScyllaDB Configuration'),
```

Comment:
I would create section specifically for Scylla version, and clarified that you need you use exactly one of those options.

Reply:
Still valid

Reply:
ah, missed this comment. Updated now